Automattic/jetpack
# Search: Fix Customizer and styling bugs introduced in 9.6

Fixes #19420, fixes #19421.

#### Changes proposed in this Pull Request:
* Fixes a bug preventing sort select in the product layout from being legible in the dark theme for IE11.
* Fixes a bug that misaligns the sort select in the product layout.
* Fixes a Customizer bug that prevents IE11 users from saving their changes within the Customizer.
* Fixes a Customizer bug that summons the search modal scrolled halfway down the page.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription.
* In IE11, enable the product layout in the Jetpack Search section. Ensure that the preview summons the Jetpack Search modal upon (1) opening the Jetpack Search section and (2) changing the search layout, if necessary.
* Summon the search modal in IE11. Ensure that the sort select is legible.
